apps/preview/next/pages/showcases/Banners/Hero.tsx

```diff
@@ -5,38 +5,28 @@ import { SfButton } from '@storefront-ui/react';
 
 export default function Hero() {
   return (
-    <div className="relative min-h-[576px]">
-      <picture>
-        <source srcSet="http://localhost:3100/@assets/hero-bg.png" media="(min-width: 768px)" />
+    <div className="relative h-dvh w-dvh">
```

Comment:
in design it should be 864px not device height, is this something decided on behind scenes?